jobs/adept_pr/roles/autotested/files/known_failures.txt

```diff
@@ -0,0 +1,8 @@
+# <NVRA>                                   <subtest name>           <human-friendly description>
+docker-1.12.6-55.gitc4618fb.el7.x86_64     docker_cli/run_volumes   should get filtered on rhelatomic7-4-docker peons
```

Comment:
You can't filter all sub-subtests by giving a subtest. You have to say `docker_cli/run_volumes/oci_umount`

Reply:
Hmmm, okay, I can live with that so long as it's documented 😀